### PR TITLE
Letter anatomy

### PR DESCRIPTION
**Pull request by AJHalford**
Addresses issue #13

### CODE REVIEW

File: elspSampleLetters.html [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+                                Endowed Chair of My Department, Impressive University<br>
+                                Society Fellow<br>
+                                Fellow of Physics<br>
+                                Fellow of Astrophysics<br>
+                                Impressive Award Medalist<br>
+                                Member of Mensa</p>
+                        </span>
+                    </div>
+                </div>
+
```
Comment (aburrell) [inline elspSampleLetters.html:113]: We need to add the good nomination letter here.

**Review by aburrell — COMMENTED** [review]
It looks like the new good letter is missing the commentary explaining why some sections are good.  @AJHalford would you like to workshop that with me this week?

**Comment by AJHalford** [comment]
Sounds good. How about Friday afternoon?

File: elspSampleLetters.html (hunk 1 above)
Comment (aburrell) [reply elspSampleLetters.html:113]: @melissaroses the `sample_letters/nomination_letters/supportive_male.docx` letter is ready to go here!

File: debug.log [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[0124/201946.819:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[0126/083958.226:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[0126/134920.486:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (aburrell) [inline debug.log]: This file should be removed and `*.log` added to the .gitignore file.

**Review by aburrell — APPROVED** [review]
I fixed anything I found wrong, and now approve this pull request.

**Comment by aburrell** [comment]
@melissaroses does this pull request also address #11 ?

**Review by aburrell — APPROVED** [review]

**Pull request merged**